From 960921a6a1cf2b0a8becfe4bc5b0a6283ee58965 Mon Sep 17 00:00:00 2001
From: Junxiao Chang <junxiao.chang@intel.com>
Date: Tue, 19 Apr 2022 10:29:56 +0800
Subject: [PATCH] drm/i915: change GuC version to 69.0.3

Except for TGL/ADL, GuC 70.0.3 were not public released. So
downgrade them to 69.0.3.

Signed-off-by: Junxiao Chang <junxiao.chang@intel.com>
---
 drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c b/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c
index 71ab97823d3c..b16ec1b18fd4 100644
--- a/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c
+++ b/drivers/gpu/drm/i915/gt/uc/intel_uc_fw.c
@@ -52,19 +52,19 @@ void intel_uc_fw_change_status(struct intel_uc_fw *uc_fw,
 #define INTEL_GUC_FIRMWARE_DEFS(fw_def, guc_def) \
 	fw_def(ALDERLAKE_P,  0, guc_def(adlp, 70, 0, 3)) \
 	fw_def(ALDERLAKE_S,  0, guc_def(tgl,  70, 0, 3)) \
-	fw_def(DG1,          0, guc_def(dg1,  70, 0, 3)) \
+	fw_def(DG1,          0, guc_def(dg1,  69, 0, 3)) \
 	fw_def(ROCKETLAKE,   0, guc_def(tgl,  70, 0, 3)) \
 	fw_def(TIGERLAKE,    0, guc_def(tgl,  70, 0, 3)) \
-	fw_def(JASPERLAKE,   0, guc_def(ehl,  70, 0, 3)) \
-	fw_def(ELKHARTLAKE,  0, guc_def(ehl,  70, 0, 3)) \
-	fw_def(ICELAKE,      0, guc_def(icl,  70, 0, 3)) \
-	fw_def(COMETLAKE,    5, guc_def(cml,  70, 0, 3)) \
-	fw_def(COMETLAKE,    0, guc_def(kbl,  70, 0, 3)) \
-	fw_def(COFFEELAKE,   0, guc_def(kbl,  70, 0, 3)) \
-	fw_def(GEMINILAKE,   0, guc_def(glk,  70, 0, 3)) \
-	fw_def(KABYLAKE,     0, guc_def(kbl,  70, 0, 3)) \
-	fw_def(BROXTON,      0, guc_def(bxt,  70, 0, 3)) \
-	fw_def(SKYLAKE,      0, guc_def(skl,  70, 0, 3))
+	fw_def(JASPERLAKE,   0, guc_def(ehl,  69, 0, 3)) \
+	fw_def(ELKHARTLAKE,  0, guc_def(ehl,  69, 0, 3)) \
+	fw_def(ICELAKE,      0, guc_def(icl,  69, 0, 3)) \
+	fw_def(COMETLAKE,    5, guc_def(cml,  69, 0, 3)) \
+	fw_def(COMETLAKE,    0, guc_def(kbl,  69, 0, 3)) \
+	fw_def(COFFEELAKE,   0, guc_def(kbl,  69, 0, 3)) \
+	fw_def(GEMINILAKE,   0, guc_def(glk,  69, 0, 3)) \
+	fw_def(KABYLAKE,     0, guc_def(kbl,  69, 0, 3)) \
+	fw_def(BROXTON,      0, guc_def(bxt,  69, 0, 3)) \
+	fw_def(SKYLAKE,      0, guc_def(skl,  69, 0, 3))
 
 #define INTEL_HUC_FIRMWARE_DEFS(fw_def, huc_def) \
 	fw_def(ALDERLAKE_P,  0, huc_def(tgl,  7, 9, 3)) \
-- 
2.25.1

